README.md

```diff
@@ -15,36 +15,36 @@ Questions or feedback? Join the [Oracle Linux Discussion](https://github.com/ora
 ### Convert CentOS to Oracle Linux
 Looking to switch from CentOS to Oracle Linux? Head over to the [centos2ol repo](/oracle/centos2ol) for a script designed to automatically switch a CentOS instance to Oracle Linux.
 
-### Run Container Images
+### Run container images
  - Download official Oracle Linux base images published on [GitHub Container Registry](https://github.com/orgs/oracle/packages/container/package/oraclelinux).
  - Download [developer-oriented images](https://github.com/orgs/oracle/packages?repo_name=docker-images) to run your application code and or connect to Oracle Database
  - Build your own container images for Oracle commercial products and Oracle sponsored open source projects using these [sample Dockerfiles](https://github.com/oracle/docker-images).
 
-### Run Vagrant Boxes and Projects
+### Vagrant boxes and projects
 - [Download Oracle Linux Vagrant boxes](https://yum.oracle.com/boxes/)
 - Explore [Vagrant projects](https://github.com/oracle/vagrant-projects) that provision Oracle and other software automatically 
 
-### Build Your Own Oracle Linux Images
+### Build your own Oracle Linux images
 Need to build custom Linux images for VirtualBox, KVM, Oracle Cloud or other clouds? See these [scripts to build Oracle Linux images](oracle-linux-image-tools)
```

Comment:
Which reminds me: we really should release the script(s) that build the Oracle Linux container images either in this repo or in oracle/container-images (or both).

Reply:
makes sense to me